scripts/launch.sh

```diff
@@ -35,4 +35,5 @@ tmux new-session     "${CMD} -d -f ${CONFIG_PATH}.0.json; sh" \; \
      splitw -h -p 50 "${CMD} -d -f ${CONFIG_PATH}.1.json; sh" \; \
      splitw -v -p 50 "${CMD} -d -f ${CONFIG_PATH}.2.json; sh" \; \
      selectp -t 0 \; \
-     splitw -v -p 50 "${CMD} -d -f ${CONFIG_PATH}.3.json; sh"
+     splitw -v -p 50 "${CMD} -d -f ${CONFIG_PATH}.3.json; sh" \; \
```

Comment:
This and the config change will break the other existing scripts. Can you create separate configs for AVSS? Can you also make this script generic by having default number of panes as 4 unless passed from the command line?